WebDriverAgentLib/Categories/XCUIElement+FBTap.m

```diff
@@ -36,6 +41,11 @@ - (BOOL)fb_tapWithError:(NSError **)error
 
 - (BOOL)fb_tapCoordinate:(CGPoint)relativeCoordinate error:(NSError **)error
 {
+  if (SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(@"11.0")) {
+    XCUICoordinate *tapCoordinate = [[XCUICoordinate alloc] initWithElement:self normalizedOffset:CGVectorMake(relativeCoordinate.x, relativeCoordinate.y)];
+    [tapCoordinate tap];
```

Comment:
It looks like coordinates inversion is still mandatory in iOS 11 if we use the current approach to perform tap action. Which means this PR has no point anymore. Shall I close it then?